### ISSUE TITLE
Staged applications: registration vs applied

### ISSUE THREAD

**Issue by warrensearle** [jac-uk/admin#2135]
We recently enabled exercises to have 'staged applications' where partial information could be requested up-front and at each stage of processing. This work needs iterating on to ensure initial 'registration' applications are not treated as having 'applied'
- [ ] Change the status when an initial application is made to REGISTERED instead of APPLIED (only when exercise configured as staged applications)
![Image](https://user-images.githubusercontent.com/15261814/172256248-2abe0329-eef2-4fee-99f2-0631cbcdc83c.png)
- [ ] Enable QTs to happen against Registered applications
- [ ] Update reports which include 'Applied' so they correctly reflect 'Registered' as well as 'Applied'

**Comment by warrensearle** [comment]
@nickaddy currently using the 'Application process' part of admin an exercise could be configured such that the full application data is not asked for until the candidate has been Recommended to SCC. Therefore it would be possible to configure an exercise where candidates are only being considered as having APPLIED after they have been recommended! Could you help figure out the best way forward here? Probably either restrict the application process or agree exactly when REGISTERED changes to APPLIED (e.g. once the candidate has completed the information asked for in 'Shortlisted' stage)


<img width="1342" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMk9tQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--528d0d71e31a83a749b98aad77ca7b469a99f498/image.png" alt="image.png" />

**Comment by nickaddy** [comment]
Yep, have a meeting with Matt later in the week to discuss this and other aspects. Will revert after that.
